app/components/Form/ProjectAnnualReportForm.tsx

```diff
@@ -162,6 +163,14 @@ const ProjectAnnualReportForm: React.FC<Props> = (props) => {
     projectRevision.emissionIntensityReportingRequirementFormChange.edges[0]
       ?.node;
 
+  const upcomingReportFormChange =
+    projectRevision.upcomingAnnualReportFormChange;
+
+  var indexOfNextAnnualReport =
+    sortedAnnualReports.findIndex(
+      (x) => x.id === upcomingReportFormChange?.id
+    ) + 1;
```

Comment:
Since this code is repeated across other forms as well, it would be great if we can make it a helper function but not a big deal.

Reply:
Also, being nitpicky :) It would possibly be better if replace `var` with `let` or `const`

Reply:
Hm, I thought we had an eslint rule for this